### PR TITLE
Add SolarSchedule event choices i18n support

### PR DESCRIPTION
**Pull request by mondeja**
`SolarSchedule` model event choices translations are not recognized by `python manage.py makemessages`. To fix this I've hardcoded solar schedules readable representations for `SolarSchedule` model event choices.

- Solar schedules are keep synchronized with `celery.schedules.solar._all_events` tuple by a unit test.
- Translations for these new values have been added to `es` and `fr` locales.
- A new migration have been added to update `SolarSchedule` model event choices.
- Minor changes in Changelog to prevent `apicheck` error message: `Changelog:11:Field list ends without a blank line; unexpected unindent.`.

### CODE REVIEW

File: django_celery_beat/migrations/0014_auto_20200609_1048.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.6 on 2020-06-09 10:48
```
Comment (auvipy) [inline django_celery_beat/migrations/0014_auto_20200609_1048.py:1]: this migration needs to be newly generated after pulling from master

**Review by auvipy — CHANGES_REQUESTED** [review]

**Comment by auvipy** [comment]
@arnau126 @asfaltboy mind to review this patch?

File: django_celery_beat/migrations/0014_auto_20200609_1048.py (hunk 1 above)
Comment (mondeja) [reply django_celery_beat/migrations/0014_auto_20200609_1048.py:1]: Do you mind if I wait for the full review of the patch?

**Review by asfaltboy — APPROVED** [review]
Seems legit, let's merge it :shipit: !

**Review by arnau126 — APPROVED** [review]

File: django_celery_beat/migrations/0014_auto_20200609_1048.py (hunk 1 above)
Comment (auvipy) [reply django_celery_beat/migrations/0014_auto_20200609_1048.py:1]: https://github.com/celery/django-celery-beat/blob/master/django_celery_beat/migrations/0014_remove_clockedschedule_enabled.py this is the last working migration which serial number seems same to this, thats why I was suggesting the remove and regenration :D

**Review by auvipy — CHANGES_REQUESTED** [review]

File: django_celery_beat/migrations/0014_auto_20200609_1048.py (hunk 1 above)
Comment (mondeja) [reply django_celery_beat/migrations/0014_auto_20200609_1048.py]: Now it's pointing to `0014_remove_clockedschedule_enabled`. Let me know if any problem persists.

File: django_celery_beat/models.py [hunk 2]
```diff
@@ -38,7 +38,17 @@
     (MICROSECONDS, _('Microsecond')),
 )
 
-SOLAR_SCHEDULES = [(x, _(x)) for x in sorted(schedules.solar._all_events)]
+SOLAR_SCHEDULES = [
```
[the remark this answers is not in the document]
Comment (mondeja) [reply django_celery_beat/models.py:41]: [Seeing the blame of the file](https://github.com/celery/django-celery-beat/blame/master/django_celery_beat/models.py) I can see that nothing related has changed and the global variable only is used in that file.

**Review by auvipy — APPROVED** [review]

**Pull request merged**